gnolang/gnosocial
# feat: profile page

This PR includes several code improvements and the development of a new page to display the user's profile and followers.

It's still on WP because I'm improving the UX/UX, but I think you should give it a try.

How to test it:
- Open the app and navigate to the search page. Search for any user and then press over any of the results. 
- The app will display a new screen that shows the number of followers and following users.
- Press the label that displays the number of followers and following users to see the list.
